### DISCUSSION TITLE
spark-doris-connector插件的一个建议

### DISCUSSION THREAD

**Discussion by young138120** [apache/doris#10764] [Ideas]
version：'org.apache.doris:spark-doris-connector-3.1_2.12:1.0.1'  
再使用插件做stream load的时候，如果出现的一些字段不匹配或者相关的问题，返回的信息里面没有带有ErrorURL信息  
直接被吞了  
![image](https://user-images.githubusercontent.com/11519151/178224072-eb914c2d-c1f4-47eb-87ec-3cea51b5d896.png)
![image](https://user-images.githubusercontent.com/11519151/178224108-c8bd4403-0bdf-43e3-9e61-1b7a79c65ce3.png)

看了下官方的文档，目前没有能够重新查到这个ErrorURL的方式或者入口耶，所以每次都需要local的模式去复现，再数据比较大的情况就会很麻烦了

**Comment by LOVEGISER** [comment]
1. Stream load is a synchronous way of importing,so returen result sync. the result can not get by `show load` command
2. doris-spark-connector and flink-doris-Connector application all have this problem
3. so I suggest Steam load result  support by `show load` command query.

**Reply by young138120** [reply]
i think it should be log the ErrorURL info ，I can trace the problem

**Reply by LOVEGISER** [reply]
These steps can query stream load record

1.  add` enable_stream_load_record=true`    in  be.conf and restart BE

2. query  stream load record by ` SHOW STREAM LOAD ` command

**Reply by JNSimba** [reply]
> i think it should be log the ErrorURL info ，I can trace the problem

Hello, thank you for your suggestion, this PR has been added to the log https://github.com/apache/doris-spark-connector/pull/44
